tests/test_plotting/test_geodesics/test_interactive.py

```diff
@@ -1,35 +1,57 @@
 import os
 from unittest import mock
 
-import astropy.units as u
+# import astropy.units as u
 import numpy as np
 import pytest
 from plotly.graph_objects import Figure
 
-from einsteinpy.bodies import Body
-from einsteinpy.coordinates import SphericalDifferential
+# from einsteinpy.bodies import Body
```

Comment:
Why is this commented out?

Reply:
These methods and imports correspond to the old API and they also use `coordinates`, which is currently unsupported. I thought, I had removed them, but apparently not. Removing these now.